commands/miscellaneous/update-command.js

```diff
@@ -1,75 +1,74 @@
-const { Command } = require("discord.js-commando");
+const { SlashCommandBuilder } = require('@discordjs/builders');
 const fs = require('fs');
-const path = require('path')
+const path = require('path');
 const got = require('got');
+// TODO: needs permissions 'MANAGE_CHANNELS', 'MANAGE_ROLES'
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('updatecommand')
```

Comment:
Your code looks good in this file! I'm just confused about what the purpose of this command is, and whether it will still be needed with slash commands / whether or not it's a good idea to have a command like this when the commands will be visible to everyone on the server. This seems to be a command for adding and removing command files, possibly for convenience on AWS?

This goes back to the question of permissions which we haven't really figured out yet, but I recall seeing that slash commands can't be hidden but can be disabled for some users?

Reply:
@seatuna when i was debugging i .setDefaultPermissions(true); i could still see it but it was in a disabled state (text greyed out) and i couldn't use it. 👍